phpstan.neon.dist

```diff
@@ -21,6 +29,7 @@ parameters:
     - includes/
   excludes_analyse:
     - includes/vendor/*
+    - includes/PhpStan/*
```

Comment:
PHPStan files should live in `tests/phpstan`, never in `includes/`

Reply:
Fixed